What's new April 2026 blog post (#2359)
withastro/astro.build: src/content/blog/whats-new-april-2026.mdx

import ImageGallery from './_astro-together-components/AstroTogetherLondonImageGallery.astro';

{/* Uncomment to load integrations in console for copy / paste 
import MonthlyTools from './_whats-new-components/MonthlyTools.astro';

<MonthlyTools date={frontmatter.startDate} />
*/}

Some exciting news from around the Astro ecosystem this month, including:

- [Updates from Mission Control](#updates-from-mission-control)
- [Latest releases](#latest-releases)
- [Astro Adopters](#astro-adopters)
- [Weird Wide Web](#weird-wide-web)
- [Cosmic Canvas](#cosmic-canvas)
- [Community updates](#community)
- [Reading List](#content)
- [Watch Party](#watch-party)
- [Tips & Tools](#tips--tools)
- [Themes & Templates](#themes--templates)
- [Site Showcase](#showcase)
- [Starlight in the wild](#starlight-in-the-wild)

Let's dive right into it!

## Updates from Mission Control 

- 💂 [Astro Together - London](https://luma.com/lzh944vx)
- 🚀 [Astro 7 alpha preview](https://astro.build/blog/astro-620/#astro-v7-alpha) an early look at Vite 8 support, the stable Rust compiler, and more.

## Latest releases

Minor releases introduce new features and improvements. Check out what's new!

> - [Astro 6.2](/blog/astro-620/) - experimental advanced routing with Hono support, an SVG optimizer API, a custom logger, and more.

Reviewer: We might need to update that if advanced routing is not ready for the release.

Author: Yes, thank you! I wrote a TODO in the PR description for exactly this reason! 🙌

  'www.thorgal.com',
  'create.michaelgrills.ca',
];

{/* Use for grids of 2 per row  */}

<div class="grid grid-cols-1 sm:grid-cols-2 gap-5">
  <ShowcaseCard site={await getEntry('showcase',  'www.wondermakers.digital')} />
  <ShowcaseCard site={await getEntry('showcase', 'pixila.net')} />
  <ShowcaseCard site={await getEntry('showcase', 'create.michaelgrills.ca',)} />
  <ShowcaseCard site={await getEntry('showcase', 'www.thorgal.com')} />
</div>

{/* Use when there are multiples of 3  */}
{/* <ShowcaseGridById ids={canvasSites} /> */}

## Community

### Astro Team News

Several members of the core team met up for Astro Together London. <Mention name="fred" />, <Mention name="matt" />, <Mention name="florian" />, <Mention name="alex" />, and <Mention name="chris" /> provided behind-the-scenes talks and live demos... while the attendees received custom swag including freeze dried astronaut ice cream to enjoy during the talks.

Check out some photos, and watch for the videos coming soon! Subscribe to the official [Astro Luma calendar](https://luma.com/astrodotbuild?period=past) to be notified of future events.

<ImageGallery
	images={[
		{
			src: 'astro-london-1.jpeg',

> 			alt: 'Bowls of various astro stickers, some that look like mission badges.',

Reviewer: nit:
```suggestion
			alt: 'Bowls of various Astro stickers, some that look like mission badges.',
```

		},
		{
			src: 'astro-london-2.jpeg',
			alt: 'A pale grey Astro totebag with a background of scattered stars with the Astro and Cloudflare logos. For the world WHOA web',
		},
		{
			src: 'astro-london-3.jpeg',
			alt: 'Single serve packages of astronaut ice cream surrounded by various Astro and Cloudflare stickers',
		},
    {
			src: 'astro-london-4.jpeg',
			alt: 'A large meeting room with the Astro logo showing on the monitors with dozens of people in chairs, waiting for the presentation to start.',
		},
    {
			src: 'astro-london-5.jpeg',

> 			alt: 'Fred introducing the the speakers in front of rows of people in the audience.',

Reviewer: ```suggestion
			alt: 'Fred introducing the speakers in front of rows of people in the audience.',
```

- [astro-dgmo](https://github.com/diagrammo/astro-dgmo#readme): Astro integration to render DGMO diagrams from fenced code blocks at build time.
- [astro-opengraph](https://github.com/altano/npm-packages/tree/main/packages/astro-opengraph) - An integration to create and manage your Open Graph images, including a dev toolbar app to preview the image and the `og` meta tags on the page.
- [@ayco/astro-resume](https://git.ayo.run/ayo/astro-resume): Utilities for serializing data from the server for use in the client.
- [@better-translate/astro](https://github.com/jralvarenga/better-translate/tree/main/packages/astro): A type-safe i18n toolkit for TypeScript.
- [astlide](https://github.com/r-hashi01/astlide): An Astro-based slide presentation framework — like Slidev, but for the Astro ecosystem.
- [astro-feedkit](https://github.com/kitschpatrol/astro-feed-kit): An Astro integration for full-content RSS, Atom, and JSON feeds.
- [astro-mdx](https://github.com/kitschpatrol/astro-mdx-kit): An Astro integration for MDX directive-to-component mapping, element overrides, and auto-imports.

### Plugins

> - [Astro Cursor Plugin](https://cursor.directory/plugins/astro-1): Cursor plugin for Astro including the official Astro Docs MCP, rules with best practises and coding standards and several skills such as add integration, content collection, create component, docs lookup and migrate.

Reviewer: typo:
```suggestion
- [Astro Cursor Plugin](https://cursor.directory/plugins/astro-1): Cursor plugin for Astro including the official Astro Docs MCP, rules with best practices and coding standards and several skills such as add integration, content collection, create component, docs lookup and migrate.
```

import ImageGallery from './_astro-together-components/AstroTogetherLondonImageGallery.astro';

{/* Uncomment to load integrations in console for copy / paste 
import MonthlyTools from './_whats-new-components/MonthlyTools.astro';

<MonthlyTools date={frontmatter.startDate} />
*/}

Some exciting news from around the Astro ecosystem this month, including:

- [Updates from Mission Control](#updates-from-mission-control)
- [Latest releases](#latest-releases)
- [Astro Adopters](#astro-adopters)
- [Weird Wide Web](#weird-wide-web)
- [Cosmic Canvas](#cosmic-canvas)
- [Community updates](#community)
- [Reading List](#content)
- [Watch Party](#watch-party)
- [Tips & Tools](#tips--tools)
- [Themes & Templates](#themes--templates)
- [Site Showcase](#showcase)
- [Starlight in the wild](#starlight-in-the-wild)

Let's dive right into it!

## Updates from Mission Control 

- 💂 [Astro Together - London](https://luma.com/lzh944vx)
- 🚀 [Astro 7 alpha preview](https://astro.build/blog/astro-620/#astro-v7-alpha) an early look at Vite 8 support, the stable Rust compiler, and more.

## Latest releases

Minor releases introduce new features and improvements. Check out what's new!

> - [Astro 6.2](/blog/astro-620/) - experimental advanced routing with Hono support, an SVG optimizer API, a custom logger, and more.

Author: ```suggestion
- [Astro 6.2](/blog/astro-620/) - an experimental custom logger with JSON output, an SVG optimizer API, a new font file URL helper, and more.
```
